server/state-selectors-generator/script-generate.js

```diff
@@ -0,0 +1,2 @@
+const stateSelectorsGenerator = require( './' );
+stateSelectorsGenerator.generateIndexFile();
```

Comment:
Actually, it isn't. 

<img src="https://cloud.githubusercontent.com/assets/77539/22162924/8b8d6b34-df2f-11e6-8ec5-0a620a855616.png" width="100px" />